### PR TITLE
Remove incorrect default text

### PR DESCRIPTION
**Pull request by Artur-**

### CODE REVIEW

File: articles/shared/guide/configuration/_index.asciidoc [hunk 1]
```diff
@@ -125,7 +125,6 @@ Uses bytecode scanning, which increases application startup time.
 Configuration name for the parameter that determines whether Vaadin should automatically register servlets needed for the application to work.
 
 `disable-xsrf-protection`::
-The default is `true`.
```
Comment (tarekoraby) [inline articles/shared/guide/configuration/_index.asciidoc:128]: Why not set the value to `false` rather than removing the note altogether?

Comment (Artur-) [reply articles/shared/guide/configuration/_index.asciidoc:128]: None of the other options state the default value this way and the text below specifies what the default is

Comment (tarekoraby) [reply articles/shared/guide/configuration/_index.asciidoc:128]: Some do, some don't 
image

**Review by tarekoraby — APPROVED** [review]

File: articles/shared/guide/configuration/_index.asciidoc (hunk 1 above)
Comment (tarekoraby) [reply articles/shared/guide/configuration/_index.asciidoc:128]: Anyways, it's your call since it's described a couple of sentences after.

**Pull request merged**